adoc/Release-Notes.adoc

```diff
@@ -117,25 +117,26 @@ Steps for a fresh install of {product} 1.5 HA:
 [source,bash]
 ----
 helm install --name susecf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA=true \
---set config.HA_strict=false --set sizing.mysql.count=1
+--set config.HA_strict=false --set sizing.mysql.count=1 --version 2.18.0
```

Comment:
Maybe change the heading "Steps for a fresh install of SUSE Cloud Application Platform 1.5 HA:" to include "Steps for a fresh install of SUSE Cloud Application Platform 1.5  **in the default HA setup**". And then be explicit that specifying `config.HA=true` and `config.HA_strict=false` will default the sizes of the roles with the exception of mysql which will be set to 1.